react-app/src/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root'),
+);
+
```

Comment:
Stuff like that would appear rather confusing. You may place your `FirstHello` component into its own file and import that separately (just as the rest of your components). Within that file, I would wrap `.createElement()` with the function that would accept parameters, which may emulate component props, This way you may use `FirstHello` as a component. Your `index.js` should be left with only main app rendered.